File: patterns/c-patterns/pattern44.c
// Pattern 44: ...

// 1*3*5
// 1*3*
// 1*3
// 1*
// 1

#include <stdio.h>
int main()
{
    int i, j;
    int n = 5;

    for (i = n; i >= 1; i--)
    {
        for (j = 1; j <= i; j++)
        {
            if (j % 2 == 0)
                printf("*");
            else
                printf("%d", j);
        }
        printf("\n");
    }
    return 0;
}

File: patterns/javascript-patterns/pattern44.js
// Pattern 44: ...

// 1*3*5
// 1*3*
// 1*3
// 1*
// 1

const n = 5;

for (let i = n; i >= 1; i--) {
    let row = '';

    for (let j = 1; j <= i; j++) {
        if (j % 2 === 0) {
            row += '*';
        } else {
            row += j;
        }
    }

    console.log(row);
}
